packages/pricing/src/providers/cachingProxy.provider.ts

```
@@ -1,13 +1,15 @@
-import { ICache, PriceCacheKey } from "@grants-stack-indexer/repository";
+import { IPricingCache } from "@grants-stack-indexer/repository";
 import { ICacheable, ILogger, TimestampMs, TokenCode } from "@grants-stack-indexer/shared";
 
-import { IPricingProvider, TokenPrice } from "../internal.js";
+import { NoClosePriceFound } from "../exceptions/noClosePriceFound.exception.js";
+import { IPricingProvider, MIN_GRANULARITY_MS, TokenPrice } from "../internal.js";
 
 type CacheResult = {
```

ESLint

[error] 'CacheResult' is defined but never used.

(@typescript-eslint/no-unused-vars)